src/components/Person/Person.jsx

```diff
@@ -1 +1,9 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ name, age, sex, isMarried, partnerName }) => (
+  <section className="Person">
```

Comment:
And destructure values inside the component
```suggestion
export const Person = ({ person }) => {
const { name, age, sex, isMarried, partnerName } = person;

return (
  <section className="Person">
```